NGitLab/ISearchClient.cs

```diff
@@ -0,0 +1,9 @@
+﻿using NGitLab.Models;
+
+namespace NGitLab
+{
+    public interface ISearchClient
+    {
+        GitLabCollectionResponse<SearchBlob> GetBlobs(SearchQuery query);
```

Comment:
I think other `GitLabCollectionResponse` methods use the `Async` suffix.
```suggestion
        GitLabCollectionResponse<SearchBlob> GetBlobsAsync(SearchQuery query);
```